src/App.tsx

```diff
@@ -1,18 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+
+const perPageOptions = [3, 5, 10, 20];
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState<number>(5);
+  const [currentPage, setCurrentPage] = useState<number>(1);
```

Comment:
it's redundant to specify the types of primitives
```suggestion
  const [perPage, setPerPage] = useState(5);
  const [currentPage, setCurrentPage] = useState(1);
```